assessments/0002-notary-project.md

```diff
@@ -0,0 +1,196 @@
+   - Who reviews documentation PRs? 
+
+### Recommendations
+
+#### Information Architecture
+
+The assumption is that V2 is the primary project moving forward, and that a Minimum Viable Product (MVP) for a website/documentation will focus on Notary V2 users.
+- Bring the nv2 cli prototype documentation into the main documentation when conosolidating the content on the website.
+- V1 Docs will need to be reworked once an information architecture is setup for the new site. The new site IA can likely be informed by the existing V1 docs.
+
+The [Service Mesh Interface website](https://smi-spec.io/) handles it's specification tab very well -- linking directly to the relevant markdown folder in GitHub, allowing website users to find the specification work where it is happening, and giving new users an opportunity to participate.
+
+
+## Contributor documentation 
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     | ✅  |     |     |
+| Beginner friendly issue backlog           |     | ✅  |     |     |     |
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+Criteria 
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work)
+- 5 = (is executed extremely well or no improvement required) |
+
+### Comments & recommendations
+
+- Most of the issues in this section are a function of not having a website yet, and the project still being in a building phase.
+  - As the website build process starts, put together a skeleton IA that can be filled in as the project grows.
+
+- There are documented community communication methods, but it could be collected and expanded during the site development process. Prometheus has a good example of a community page: https://prometheus.io/community/. 
+
+- Regarding beginner friendliness, consider processes/documents for:
+   - Triaging docs issues
+   - Clearly marking a way for new contributors to make code or documentation contributions (i.e. a “good first issue” label), and define what makes a good first issue
+   - Set up a process for maintaining/pruning stale issues
+   - New contributors/your first contribution
+   - Letting new users know where to get help
+
+- Complete the governance work started here: https://github.com/notaryproject/notaryproject/issues/55 https://github.com/notaryproject/notaryproject/pull/78, and add it to a governance page (or section) on the website.
+
+
+## Final Recommendations
+
+Is it possible plan documentation along side the V2 development? Will the documentation be able to get its structure from the specifications being developed?
+
+### Website
```

Comment:
I love the checklist format of this section. It's concise and actionable.

Reply:
+1! This is an awesome way to do this part. I'm glad you did this, @nate-double-u - it's a good model of what we can do with these going forward.

Reply:
Thanks -- should I refactor and use this format in other areas of this document?